perun-beans/src/main/java/cz/metacentrum/perun/core/api/exceptions/WrongRangeOfCountException.java

```diff
@@ -8,7 +8,7 @@
 /**
  * Exception, which is using for bad range of messages count for audit messages.
  *
- * @see cz.metacentrum.perun.core.api.exceptions.rt.InternalErrorException
+ * @see cz.metacentrum.perun.core.api.exceptions.rt.InternalErrorRuntimeException
```

Comment:
You can probably remove this. It's copy/paste error from some other exception plus we will most likely remove runtime versions of our exceptions since they are unused.